pkg/volume/aws_ebs/aws_util.go

```diff
@@ -91,12 +92,19 @@ func (util *AWSDiskUtil) CreateVolume(c *awsElasticBlockStoreProvisioner) (aws.K
 	}
 	// Apply Parameters (case-insensitive). We leave validation of
 	// the values to the cloud provider.
+	zonePresent := false
+	zonesPresent := false
+	var calculateZonesParams volume.CalculateSetOfZonesParams
 	for k, v := range c.options.Parameters {
 		switch strings.ToLower(k) {
 		case "type":
 			volumeOptions.VolumeType = v
 		case "zone":
-			volumeOptions.AvailabilityZone = v
+			zonePresent = true
+			calculateZonesParams.StorageClassZones = v
```

Comment:
Yes, it will interpret the `zone` parameter as a comma separated list of zones. Assume, the documentation won't mention this extra functionality. Is it a problem considering the `zone` parameter will be removed in (near) future because it is being replaced by the `zones` parameter?